pkg/sql/importer/import_processor.go

```diff
@@ -152,6 +154,11 @@ func newReadImportDataProcessor(
 	}
 	if err := cp.Init(ctx, cp, post, csvOutputTypes, flowCtx, processorID, output, nil, /* memMonitor */
 		execinfra.ProcStateOpts{
+			TrailingMetaCallback: func() []execinfrapb.ProducerMetadata {
+				log.Info(ctx, "TrailingMetaCallback")
+				cp.close()
+				return nil
+			},
```

Comment:
restore calls consumer closed in their call back. Should import do this as well?

Reply:
ah nvm. i'm being dumb.